spring-cloud-consul-discovery/src/main/java/org/springframework/cloud/consul/discovery/ConsulDiscoveryProperties.java

```diff
@@ -160,6 +164,12 @@
 	 */
 	private boolean queryPassing = false;
 
+	/**
+	 * List of status considered as healthy. Contains "passing" by default.
+	 */
+	private Set<Check.CheckStatus> statusConsideredAsHealthy = new HashSet(
```

Comment:
We should document the new property

Reply:
You can add the explanation of this feature to `docs/src/main/asciidoc/spring-cloud-consul.adoc`.
`docs/src/main/asciidoc/_configprops.adoc` will be automatically generated for you so you don't need to make any changes here.